index.html

```diff
@@ -104,15 +104,16 @@
         const keys = Object.keys(fields);
 
         keys.forEach(key => {
-          const value = fields[key].getValue();
-
-          if (typeof value === 'string') {
-            while ((match = URL_REGEXP.exec(value)) !== null) {
-              result.push(match[0])
+          const translations = Object.keys(fields[key]._fieldLocales)
```

Comment:
I missed that in when commenting the issue but we probably shouldn't go for `_locales` (underscore props usually mean "dont touch them") but rather with `entry.field.locales` (https://github.com/contentful/ui-extensions-sdk/blob/master/docs/ui-extensions-sdk-frontend.md#entryfieldsid-field).

And then use `getValue` with the locale.

Reply:
ouch, totally right.
I couldn't figure a way without using the private prop.

nice catch! 👌